allennlp/data/data_loaders/multitask_data_loader.py

```diff
@@ -309,9 +247,9 @@ def _get_instances_for_epoch(self) -> Dict[str, Iterable[Instance]]:
 
     def _make_data_loader(self, key: str) -> MultiProcessDataLoader:
```

Comment:
Line 259-262 can be removed. Those params to the data loader don't exist anymore.

Reply:
done